### ISSUE TITLE
Genre Detail page should contain a "Chants by genre" list

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/CantusDB#722]
Steps:
1. Pick a manuscript;
2. Pick a folio;
3. Click on any "Genre" abbreviation hyperlink (you can only do this in NewCantus).

OR
1. Pick a feast;
2. Click on any "Genre" abbreviation hyperlink.

On OldCantus, this leads you to a page about that genre, which contains a list of all the chants that belong to that genre. In NewCantus, the page only has the genre name, and nothing else.

Demo (using the feast route in OldCantus, then NewCantus):

https://github.com/DDMAL/CantusDB/assets/83373378/15c2d683-f9a7-45b0-892a-752b8fcd886a

**Comment by annamorphism** [comment]
Wild--when I tried finding this page for #498 I couldn't find it, but now I can (maybe it only shows up some places and not others?)

**Comment by JoyfulGen** [comment]
My bad! I see this has been dealt with. Closing issue.

**Comment by annamorphism** [comment]
No,  I think you were right to point this out, because there are actually several things that could or should be done here:
-NewCantus leads to this "Genre" pages in places where OldCantus does not. These should probably go away. 
Then either:
-Leave those places where you can get to the genre page, but have something show when you get there (a list of all genres, a set of samples,whatever--not a blank page) or
-Eliminate the links to the genre pages entirely

**Comment by jacobdgm** [comment]
Let me see if I understand what's going on. There are two distinct differences, right?:

1. There are several places in NewCantus where we display links to Genre Detail pages, and OldCantus doesn't contain links in the corresponding places
    - I've been adding links to genre/office/feast detail pages as I come across places where genres/offices/feasts are displayed. I'd been thinking of this as me being helpful, but if you think it would be best @annamorphism, we can go back and get rid of them.  (I don't think this would be a good use of time, though - greater connectivity/more links is generally better in my philosophy, as long as they go where you expect them to)
2. On the Genre Detail page on OldCantus, there is a list of chants by genre. NewCantus's Genre Detail pages don't have this list
    - if I recall, this list was not on OldCantus until recently, which was why I thought #536 was justified. But if OldCantus has this, we can put it back in - with what I know now, there are probably optimizations we can apply to make the page load faster than it did. I hope it's fine if I rename this issue to focus on this latter point.

**Comment by annamorphism** [comment]
Yes, there are two things. Issue 2 was confusing me, because I thought there was no list of chants by genre on OldCantus, and I wasn't sure if I simply hadn't come across it or if it was newly added or what. It doesn't seem to me like a very useful page in its current incarnation,  so it's probably worth checking if this page is coming or going, so to speak.

**Comment by jacobdgm** [comment]
In an email discussion with Debra, she expressed that it is fine if we leave this without the list of Cantus IDs for the time being. We may want to implement it eventually if we can make the page load quickly, but it's a low priority for now.
